lib/maze/client/appium/bs_client.rb

```diff
@@ -67,7 +67,7 @@ def device_capabilities
 
         def log_run_intro
           # Log a link to the BrowserStack session search dashboard
-          url = "https://app-automate.browserstack.com/dashboard/v2/search?query=#{Maze.run_uuid}&type=builds"
+          url = "https://automate.browserstack.com/projects/#{ENV['BUILDKITE_PIPELINE_NAME']}/builds/#{Maze.run_uuid}/1?tab=tests"
```

Comment:
This won't work when running locally as `BUILDKITE_PIPELINE_NAME` won't be set.  What you could do though is make use of the existing method to get the project, something like `project_name_capabilities[:project]`. 

It would make sense to use that for the build as well, to future proof it and stop the link getting out of sync with what we actually set.

Reply:
Ah yes, I forget about MR running locally.